Update/some_005.txt

```diff
@@ -729,7 +729,7 @@ void main()
 //rじゃあ、本当は千紗登ちゃんは全然喜んでなくて、…だから、珠子ちゃんも冷たくて、それどころか、私のこと、…嫌なやつだって、思って……？
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "じゃあ、本当は千紗登ちゃんは全然喜んでなくて、…だから、珠子ちゃんも冷たくて、それどころか、私のこと、…嫌なやつだって、思って……？",
-			NULL, "Well, Chisato-chan wouldn't be pleased at all... and that would cause Tamako-chan to start acting cold, so am I... the bad guy here......?", Line_Normal);
+			NULL, "Then Chisato-chan wasn't actually happy for me... Which would cause Tamako-chan to start acting cold, ... and hate me......?", Line_Normal);
```

Comment:
```suggestion
			NULL, "Then Chisato-chan wasn't actually happy for me... Which would cause Tamako-chan to start acting cold, ...and hate me......?", Line_Normal);
```